PosteAPI/views.py

```diff
@@ -83,6 +81,29 @@ def post(self, request, *args, **kwargs):
                 {"result": {"success": False, "errors": serializer.errors}},
                 status=status.HTTP_400_BAD_REQUEST,
             )
+          
+    def update(self, request, *args, **kwargs):
```

Comment:
This method was built on LoginView; was this intended for DataView, which is called from the endpoint?